js/views/file_input_view.js

```diff
@@ -255,10 +258,6 @@
         },
 
         clearForm: function() {
-            if (this.oUrl) {
-                URL.revokeObjectURL(this.oUrl);
-                this.oUrl = null;
-            }
```

Comment:
FWIW, `loadImage` internally uses `createObjectURL` and then revokes it. Since we use data URL we should be fine but let me know if this has caused troubles in the past.

Reply:
Maybe you can talk through what elements were in memory during the process before your changes and after the changes? Feels like there is still some cleanup to do here, but I'm not in the details.

Reply:
I looked through it and my change should have been fine as we end up with a data URL (`string`) whose reference we preserve.